ingest/src/main/java/com/connexta/ingest/config/ApplicationConfiguration.java

```diff
@@ -18,8 +20,27 @@
 @EnableAutoConfiguration
 @Profile("default")
 public class ApplicationConfiguration {
```

Comment:
✏️ Rename the class to describe what it configures: Ex. `ClientConfiguration`

Reply:
Right now, it contains the configs for the TransformClient, StoreClient, and LoggingFilter. I think we can keep it as ApplicationConfiguration and split later if we need to.